x/distribution/keeper/migrations.go

```diff
@@ -0,0 +1,20 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	v042 "github.com/cosmos/cosmos-sdk/x/distribution/legacy/v042"
+)
+
+// MigrationKeeper is an interface that the keeper implements for handling
+// in-place store migrations.
+type MigrationKeeper interface {
+	// Migrate1 migrates the store from version 1 to 2.
+	Migrate1(ctx sdk.Context) error
+}
+
+var _ MigrationKeeper = (*Keeper)(nil)
+
+// Migrate1 implements MigrationKeeper.Migrate1 method.
+func (keeper Keeper) Migrate1(ctx sdk.Context) error {
+	return v042.MigrateStore(ctx, keeper.storeKey)
+}
```

Comment:
> func (ctx) {return v42.MigrateStore(ctx, keeper.storeKey)

This would be ideal, but we can't access private `keeper.storeKey` there.

OK for the Migrator way.

Reply:
828a41b